doc/man3/EVP_PKEY_type.md

```diff
@@ -0,0 +1,43 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_type,
+EVP_PKEY_id,
+EVP_PKEY_base_id,
+EVP_PKEY_set_type,
+EVP_PKEY_set_type_str,
+EVP_PKEY_set_alias_type,
+- examine and manipulate type of public key
```

Comment:
Perhaps: "examine and manipulate the type of a key"? EVP_PKEY could be a private/public key pair or just a public key.